### ISSUE TITLE
psol download in more compressed formats ?

### ISSUE THREAD

**Issue by centminmod** [pagespeed/ngx_pagespeed#1093]
For https://dl.google.com/dl/page-speed/psol/${NPS_VERSION}.tar.gz some servers have slow download speeds even as low as 300-650KB/s and at 112MB tar.gz file it can take a while to download. A suggestion is to offer the PSOL in other more compressed formats for smaller download size like xz or bzip2.

this is with xz and bzip2 at level 4 compression and pigz at level 11 compression compared to original 112MB tar.gz file

```
-rw-r--r-- 1 root   root 511M Jan 13 15:50 1.10.33.2.tar
-rw-r--r-- 1 root   root 104M Jan 13 15:50 1.10.33.2.tar.bz2
-rw-r--r-- 1 root   root 112M Dec  1 20:59 1.10.33.2.tar.gz
-rw-r--r-- 1 root   root  69M Jan 13 15:50 1.10.33.2.tar.xz
```

xz format at level 6 compression

```
ls -Alh
total 1.3G
-rw-r--r-- 1 root   root 511M Jan 13 15:50 1.10.33.2.pigz.tar
-rw-r--r-- 1 root   root 511M Jan 13 15:50 1.10.33.2.tar
-rw-r--r-- 1 root   root 104M Jan 13 15:50 1.10.33.2.tar.bz2
-rw-r--r-- 1 root   root 112M Dec  1 20:59 1.10.33.2.tar.gz
-rw-r--r-- 1 root   root  64M Jan 13 15:50 1.10.33.2.tar.xz
```

**Comment by crowell** [comment]
currently we're doing gzip at compression level 9, xz is probably a good idea for future releases if it can get us down to ~1/2 the size. thanks for the info :+1:

**Comment by centminmod** [comment]
yeah just trying to speed up my lemp install times and this probably takes up to 1/5th of the total time :)

another idea is split up the download into 2 to 4 parts, folks can then download them in parallel too - so 64MB/4 = 16MB per part

**Comment by crowell** [comment]
for the parallel connections, you can use a tool like axel http://manpages.ubuntu.com/manpages/hardy/man1/axel.1.html

and do something like 

axel -n 4 url

**Comment by centminmod** [comment]
yeah i use to use axel but seems broken on centos in the past .. probably need to take another look at it as it seems to work now
